app/services/update_user.rb

```diff
@@ -33,7 +33,24 @@ def update_sam
     reckoner.delay.set!
   end
 
+  def parsed_user_params
+    user_params.each_with_object({ }) do |(key, value), hash|
+      converted_value = nil_not_empty_string(value)
+      hash[key] = converted_value
+    end
+  end
```

Comment:
Also, if you need more advanced normalization, such as making sure the input doesn't have extra spaces, I recommend this gem: https://github.com/holli/auto_strip_attributes